external/source/exploits/CVE-2023-29336/CVE-2023-29336/CVE-2023-29336.cpp

```diff
@@ -0,0 +1,693 @@
+// writeup link:  https://www.numencyber.com/cve-2023-29336-win32k-analysis/
+// Portions of this code appears to be taken from https://github.com/leeqwind/HolicPOC/blob/58705c366b5acfe366cab5ad535a710775ba9897/windows/win32k/CVE-2017-0263/x86.cpp

```

Comment:
This code is under MIT license: https://github.com/leeqwind/HolicPOC/blob/58705c366b5acfe366cab5ad535a710775ba9897/LICENSE